candid/evm_rpc.did

```diff
@@ -234,13 +234,9 @@ type TransactionReceipt = record {
   gasUsed : nat;
 };
 type UpdateProviderArgs = record {
-  cyclesPerCall : opt nat64;
+  providerId : nat64;
   credentialPath : opt text;
-  hostname : opt text;
   credentialHeaders : opt vec HttpHeader;
-  primary : opt bool;
-  cyclesPerMessageByte : opt nat64;
```

Comment:
Updated to only remove the ability to change the hostname. I suppose it makes sense to mutate the cycles costs rather than creating new providers, since otherwise callers could just use the original provider for lower costs.